GoogleForCreators/web-stories-wp
# Fixed line-height css generation

## Summary

Make sure generated output is single-line - no line-breaks allowed in CSS in these places.

## To-do


## QA Instructions

To validate that the bug has been fixed:
1. Add a text field
1. Add a line-break inside the text field
1. Set background mode to highlight
1. Set line-height to 3
1. Click the preview button
1. Observe line-height correctly set to 3 and all text is visible
---



Fixes #5200
